src/SFA.DAS.EmployerDemand/Application/ProviderInterest/Commands/CreateProviderInterests/CreateProviderInterestsCommandHandler.cs

```diff
@@ -12,18 +15,46 @@ namespace SFA.DAS.EmployerDemand.Application.ProviderInterest.Commands.CreatePro
     public class CreateProviderInterestsCommandHandler: IRequestHandler<CreateProviderInterestsCommand, Guid>
     {
         private readonly IEmployerDemandApiClient<EmployerDemandApiConfiguration> _apiClient;
+        private readonly INotificationService _notificationService;
 
         public CreateProviderInterestsCommandHandler(
-            IEmployerDemandApiClient<EmployerDemandApiConfiguration> apiClient)
+            IEmployerDemandApiClient<EmployerDemandApiConfiguration> apiClient,
+            INotificationService notificationService)
         {
             _apiClient = apiClient;
+            _notificationService = notificationService;
         }
 
         public async Task<Guid> Handle(CreateProviderInterestsCommand request, CancellationToken cancellationToken)
         {
             var result = await _apiClient.PostWithResponseCode<PostCreateProviderInterestsResponse>(
                 new PostCreateProviderInterestsRequest(request));
 
+            if (result.StatusCode == System.Net.HttpStatusCode.Created)
```

Comment:
Is the change done in the employer demand API to differentiate between created and accepted?